src/plugins/dashboard/public/dashboard_app/_dashboard_app_strings.ts

```diff
@@ -28,7 +28,7 @@ export const dashboardReadonlyBadge = {
 export const dashboardManagedBadge = {
   getTooltip: () =>
     i18n.translate('dashboard.badge.managed.tooltip', {
-      defaultMessage: 'This dashboard is system managed. Clone this dashboard to make changes.',
+      defaultMessage: 'This dashboard is system managed. Clone it to make changes.',
```

Comment:
Wouldn't be more consistent with other wording to say, `Elastic manages this dashboard. Clone it to make changes.`?

Reply:
@amyjtechwriter this suggestion makes sense to me. WDYT?

Reply:
Yes that's more consistent. Let's change it to Tims suggestion! Thank you @drewdaemon and @tsullivan

Reply:
Changed in https://github.com/elastic/kibana/pull/178055/commits/16726cacd2966fb056afd93495e10f296c692da8